Compiler/boot/LoadCompilerSources.mos

```diff
@@ -324,6 +327,7 @@ if true then /* Suppress output */
     "../Util/HashTableExpToIndexExp.mo",
     "../Util/HashTableCrToExpSourceTpl.mo",
     "../Util/HashTableCrToCrEqLst.mo",
+    "../Util/HashTableSM1.mo",
     "../Util/HashTableCrToUnit.mo",
```

Comment:
This causes the error: 
``
[/OpenModelica/OMCompiler/Compiler/boot/build/InstStateMachineUtil.interface.mo:1:1-81:25:writable] Error: Failed to get dependencies for package InstStateMachineUtil. Perhaps there is an import to a non-existing package.
``